node-module-restore-cache/action.yml

```diff
@@ -0,0 +1,13 @@
+name: actions/node-module-restore-cache
+description: "Restore node_modules from cache"
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Restore "node_modules" from cache
+      uses: martijnhols/actions-cache/restore@v3
+      with:
+        path: "**/node_modules"
+        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
```

Comment:
Ideally we could specify the install command as an input, and have this be the default. 
The reason is easier switching to npm/pnpm if we ever need to.

Reply:
Moved to input with default